sql/item_mods.sql

```diff
@@ -35057,6 +35057,522 @@ INSERT INTO `item_mods` VALUES (28670,1,1);
 INSERT INTO `item_mods` VALUES (28757,23,14);
 INSERT INTO `item_mods` VALUES (28757,27,-3);
 /*!40000 ALTER TABLE `item_mods` ENABLE KEYS */;

```

Comment:
Don't put new lines after that `ALTER TABLE` line

Just put the ID's in order, I give up on commenting because everyone is goddamn ignoring me.
> -- ORDER BY:  `itemId`,`modId`